### ISSUE TITLE
Rename the columns "Unnamed 0:" and "Unnamed 0.1:" in get_multi_criteria_analysis

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#2711]
The script `get_multi_criteria_analysis` has two output columns that are not named. They are index columns, but should either be dropped or renamed.

This requires also updating `schemas.yml` to reflect the changes made when fixing this issue.

```
'Unnamed: 0':
        description: Index
        type: int
        unit: '[-]'
        values: '[0...n]'
'Unnamed: 0.1':
        description: Index
        type: int
        unit: '[-]'
        values: '[0.0...n]'
```

**Comment by JIMENOFONSECA** [comment]
@daren-thomas I can help, but can you fix the schemas to include the colours so we get rid of naming.csv, please? In Cooling Singapore we are trying to add new plots...but the tasks is becoming quite repetitive, having to do the naming of variables in schemas and naming csv twice. see #2718

**Comment by daren-thomas** [comment]
@JIMENOFONSECA see #2680 - we can't get rid of `naming.csv` quite yet as it does not do the same thing as `schemas.yml` and therefore can't be merged quite yet.

- `schemas.yml` describes input and output files and their schema (columns).
- `naming.csv` describes variables used in plots and their colors, legend text etc.

While there is some overlap, `naming.csv` refers to values that aren't part of the output files, but instead derived from them (e.g. aggregates). The current plan is to describe a plot as depending on a set of pre-computation steps that function similar to other cea scripts and have input- and output files - the plots would know how to run those scripts automatically when being plotted. This will replace some portions of the `.cache` as well as simplifying how to present the plot raw data to the user. This will allow us to define the plot variables in `schemas.yml` and thus get rid of `naming.csv`.

(the above text is being migrated to #2680 for future reference)

I'm a bit surprised you have new variables in `schemas.yml`?

**Comment by JIMENOFONSECA** [comment]
Let me clarify.

Every output variable in CEA goes directly into the plots without postprocessing the naming.  I personally spent months making this possible.

If there is this case in some plots. Those plots are just wrong and should we reworked to avoid unnecessary postprocessing.

In that sense the schemas should replace naming.csv 100%. And they are able to. It is just annoying to have to put new schemas and naming every time we
modify/add/remove a variable from CEA

The schemas are just lacking the colour entry and a connection to the plots.

On 30 Apr 2020, 20:45 +0800, Daren Thomas <notifications@github.com>, wrote:

@JIMENOFONSECA<https://github.com/JIMENOFONSECA> see #2680<https://github.com/architecture-building-systems/CityEnergyAnalyst/issues/2680> - we can't get rid of naming.csv quite yet as it does not do the same thing as schemas.yml and therefore can't be merged quite yet.

  *   schemas.yml describes input and output files and their schema (columns).
  *   naming.csv describes variables used in plots and their colors, legend text etc.

While there is some overlap, naming.csv refers to values that aren't part of the output files, but instead derived from them (e.g. aggregates). The current plan is to describe a plot as depending on a set of pre-computation steps that function similar to other cea scripts and have input- and output files - the plots would know how to run those scripts automatically when being plotted. This will replace some portions of the .cache as well as simplifying how to present the plot raw data to the user. This will allow us to define the plot variables in schemas.yml and thus get rid of naming.csv.

(the above text is being migrated to #2680<https://github.com/architecture-building-systems/CityEnergyAnalyst/issues/2680> for future reference)

I'm a bit surprised you have new variables in schemas.yml?

**Comment by JIMENOFONSECA** [comment]
solved in #2728
